user_docs/en/userGuide.t2t

```diff
@@ -70,7 +70,7 @@ For further details, you can [view the full licence. http://www.gnu.org/licenses
 -
 
 + Getting and Setting Up NVDA +[GettingAndSettingUpNVDA]
-If you have not yet got a copy of NVDA, you can download it from [www.nvaccess.org NVDA_URL].
+If you have not yet got a copy of NVDA, you can download it from [NVAccess website NVDA_URL].

```

Comment:
A "the" is missing. Also note that there should be a space between NV and Access: "from the NV Access website." 

```suggestion
If you have not yet got a copy of NVDA, you can download it from the [NVAccess website NVDA_URL].
```